src/NuGet.Core/NuGet.Protocol/NuGet.Protocol.csproj

```
@@ -39,17 +39,30 @@
     <Compile Include="$(SharedDirectory)\HashCodeCombiner.cs" />
     <Compile Include="$(SharedDirectory)\NoAllocEnumerateExtensions.cs" />
     <Compile Include="$(SharedDirectory)\NullableAttributes.cs" />
+    <Compile Include="$(SharedDirectory)\SharedStrings.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>$(SharedDirectory)\SharedStrings.resx</DependentUpon>
```

Checking out this PR's branch, looking at Solution Explorer, SharedStrings.Designer.cs isn't nested under SharedStrings.resx. And editing the resx, VS edits ths csproj with the below change, which fixes the file nesting

```suggestion
      <DependentUpon>SharedStrings.resx</DependentUpon>
```